### DISCUSSION TITLE
Feature Request: Options instead of Default

### DISCUSSION THREAD

**Discussion by circa1741** [tteck/Proxmox#93] [Ideas]
It would be nice to make the following as options instead of default:

1. Make Adminer an installation option for MariaDb instead of installing Adminer by default.
2. Make WGDashboard an installation option for WireGuard instead of installing WGDashboard by default.
3. Make Webmin an installation option for the LXC instead of a separate additional installation.

For example:

     1) Privileged 
     2) Unprivileged 
Please choose an Install Method and hit enter or x to exit.

     1) Use Automatic Login 
     2) Use Password (changeme) 
Please choose a Password Type and hit enter or x to exit.

     1) Automatic DHCP 
     2) Manual DHCP 
Please choose a DHCP Type and hit enter or x to exit.

     1) Graphical Interface
     2) Command Line Interface 
Please choose an Interface Type and hit enter or x to exit.

It would then also be nice to be able to install or uninstall the Graphical Interface after the initial LXC installation. The update would then be able to determine if the Graphical Interface was installed or not.

**Comment by tteck** [comment]
With V2 scripts, it's really difficult to carry options from the initial script into the install script. So, an additional menu would be needed after the LXC starts for install options

**Comment by tteck** [comment]
Testing this with the Docker LXC script, with an option to add Portainer.

**Reply by circa1741** [reply]
Yes, I just saw the Docker LXC with an option to add Portainer. I hope you get it working.

**Reply by tteck** [reply]
I can get it to work, but do I want to go that route. 

If I make it an option, say to add WGDashboard and the user doesn't select it, but later decides they want it and doesn't want to create another LXC and loose their config, I would need to create a script to add it to their existing LXC. Where as now, if you don't need that feature, don't use it, but later you can.  

I'm just not sure it's worth it.

**Reply by circa1741** [reply]
No problem. Thank you.
